revolv/static/js/base/project.js

```diff
@@ -0,0 +1,68 @@
+$(document).ready(function () {
+	drawPercentagePartialCircle()	
+
+});
+
+var drawPercentagePartialCircle = function() {
+    var radius = 100,
+        padding = 10,
+        radians = 2 * Math.PI;
+
+    if ($(document).width() > 200 && $(document).width() < 800) {
+        radius = 48;
+        //radius = $(document).width()*0.16;
+    }
+
+    var dimension = (2 * radius) + (2 * padding);
+
+    var svg = d3.select(".internal-graphics-container")
+        .attr("width", dimension)
+        .attr("height", dimension)
+        .append("g");
+
+    var circleGrouping = svg.append("g").attr("class", "project-badge-circle-grouping");
+    var partialGrouping = svg.append("g").attr("class", "project-badge-partial-grouping");
+
+    drawD3PartialTriangle(circleGrouping, ["project-badge-circle"], radius, padding, 1);
+
+    console.log(window.project)
+    
+    //this line will get actual partial completeness
+    var partialCompleteness = $(".storage-div").attr("id");
+
+    //use this for test purposes so you can see the progress bar
+    // var partialCompleteness = 0.7;
+    
+    if (partialCompleteness <= 0.01 && partialCompleteness !== 0.0) partialCompleteness = 0.02;
+    drawD3PartialTriangle(partialGrouping, ["project-badge-line"], radius, padding, partialCompleteness);
+}
+
+var drawD3PartialTriangle = function (destination, classes, radius, padding, partial) {
```

Comment:
This function is now duplicated from home.js, so we should extract it into a utility file so as to not repeat code. Also, the name draw3dPartialTriangle is kind of a misnomer, since it actually draws a circle...sorry about that, that's my fault lol. Do you mind renaming it and also writing a docstring?